binding/SkiaSharp.NativeAssets.Linux/SkiaSharp.NativeAssets.Linux.csproj

```diff
@@ -10,11 +10,13 @@
     <PackageFile Include="..\..\output\native\linux\x86\libSkiaSharp*" PackagePath="runtimes\linux-x86\native\%(Filename)%(Extension)" />
```

Comment:
Same as above, are you able to also update this file to be `libSkiaSharp*.so` instead of `libSkiaSharp*`?